ush/templates/FV3LAM_wflow.xml

```diff
@@ -813,15 +818,58 @@ MODULES_RUN_TASK_FP script.
     </dependency>
 
   </task>
+
 <!--
 ************************************************************************
 ************************************************************************
 -->
   <metatask name="&RUN_POST_TN;{{ uscore_ensmem_name }}">
 
-    <var name="fhr"> {% for h in range(0, fcst_len_hrs+1) %}{{ " %03d" % h  }}{% endfor %} </var>
+    <var name="fhr"> {% for h in range(0, postproc_len_hrs+1) %}{{ " %03d" % h  }}{% endfor %} </var>
 
-    <task name="&RUN_POST_TN;{{ uscore_ensmem_name }}_f#fhr#" cycledefs="forecast" maxtries="{{ maxtries_run_post }}">
+    <task name="&RUN_POST_TN;{{ uscore_ensmem_name }}_f#fhr#" cycledefs="postproc" maxtries="{{ maxtries_run_post }}">
+
+      &RSRV_DEFAULT;
+      &WALL_LIMIT_POST;
+      <command>&LOAD_MODULES_RUN_TASK_FP; "&RUN_POST_TN;" "&JOBSDIR;/JREGIONAL_RUN_POST"</command>
+      <nodes>{{ nnodes_run_post }}:ppn={{ ppn_run_post }}</nodes>
+      <walltime>{{ wtime_run_post }}</walltime>
+      <nodesize>&NCORES_PER_NODE;</nodesize>
+      <jobname>&TAG;_&RUN_POST_TN;{{ uscore_ensmem_name }}_f#fhr#</jobname>
+      <join><cyclestr>&LOGDIR;/&RUN_POST_TN;{{ uscore_ensmem_name }}_f#fhr#_@Y@m@d@H.log</cyclestr></join>
+
+      <envar><name>GLOBAL_VAR_DEFNS_FP</name><value>&GLOBAL_VAR_DEFNS_FP;</value></envar>
+      <envar><name>PDY</name><value><cyclestr>@Y@m@d</cyclestr></value></envar>
+      <envar><name>CDATE</name><value><cyclestr>@Y@m@d@H</cyclestr></value></envar>
+      <envar><name>CYCLE_DIR</name><value><cyclestr>&CYCLE_BASEDIR;/@Y@m@d@H</cyclestr></value></envar>
+      <envar><name>SLASH_ENSMEM_SUBDIR</name><value><cyclestr>{{ slash_ensmem_subdir }}</cyclestr></value></envar>
+      <envar><name>cyc</name><value><cyclestr>@H</cyclestr></value></envar>
+      <envar><name>fhr</name><value>#fhr#</value></envar>
+
+      <dependency>
+        <and>
+          <datadep age="05:00"><cyclestr>&CYCLE_BASEDIR;/@Y@m@d@H/fcst_fv3lam{{ slash_ensmem_subdir }}/dynf#fhr#.nc</cyclestr></datadep>
+          <datadep age="05:00"><cyclestr>&CYCLE_BASEDIR;/@Y@m@d@H/fcst_fv3lam{{ slash_ensmem_subdir }}/phyf#fhr#.nc</cyclestr></datadep>
+        </and>
+      </dependency>
+
+    </task>
+
+  </metatask>
+
+{% if do_ensemble %}
+  </metatask>
+{% endif %}
+
+<!--
+************************************************************************
+************************************************************************
+-->
+  <metatask name="&RUN_POST_LONG_TN;{{ uscore_ensmem_name }}">
```

Comment:
In order to have two lengths, is having two tasks (run_post and run_post_long) required?  I know we has lots of similar tasks for the HRRR, so maybe it is required, but it will make the workflows grow really long, like the HRRR, which is not ideal.

Reply:
If we have to use two tasks because each task has its own dependences. Otherwise, we can use parameters inside scripts to control.

Reply:
Hi Ming, we are able to combine normal and long post/ncl tasks into one post/ncl task. I think the reason that they cannot be combined at this PR is due to the fact that normal and post uses different cycledef and rocoto xml don't natively support this kind of dynamic metatasks.  I figured out a workaround as follows:
```
    <var name="fhr"> {% for h in range(0, postproc_long_len_hrs+1) %}{{ " %03d" % h  }}{% endfor %} </var>
    <var name="type"> {% for h in range(0, postproc_len_hrs+1) %}{{ " normal" }}{% endfor %} {% for h in range(postproc_len_hrs+1, postproc_long_len_hrs+1) %}{{ " long"  }}{% endfor %} </var>

    <task name="&RUN_POST_LONG_TN;{{ uscore_ensmem_name }}_f#fhr#" cycledefs="postproc_#type#" "{{ maxtries_run_post }}">
```
It will let the long post/ncl tasks to use the postproc_long cycledef.

Reply:
FYI, I have a testing xml at:
/mnt/lfs4/BMC/wrfruc/gge/rtma/ufs-srweather-app/regional_workflow/ush
where test.xml is before jinja processing and test_new.xml is the final xml.

Reply:
For this change, we need to define the following cycledef:
```
POSTPROC_CYCLEDEF="00 00-23 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
POSTPROC_LONG_CYCLEDEF="00 03,09,15,21 ${CYCLEDAY} ${CYCLEMONTH} 2021 *"
```
and change
<cycledef group="postproc"> {{ postproc_cycledef }} </cycledef>
to
<cycledef group="postproc_normal"> {{ postproc_cycledef }} </cycledef>

Reply:
Guoqing, this looks like a solution. I will try this one.

Reply:
Tried  the solution and it works very well. I updated the scripts for all runs.